packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -50,6 +51,14 @@ export interface IHotkeyProps {
     group?: string;
 
     /**
+     * The platform type on which the application is running. This informs how keys will be named
+     * within the hotkey dialog. For example, on `PlatformType.MAC`, the hotkey dialog will refer to
+     * the "meta" key as <kbd>Cmd</kbd>, while on Windows, the hotkey dialog will refer to the
+     * "meta" key as <kbd>Ctrl</kbd>.
+     */
+    platformType?: PlatformType;
```

Comment:
I would prefer to push global-ish props as high as you can in the component tree and avoid adding `context` to Blueprint for now.